clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -310,6 +317,7 @@ export const CustomSelect = ({
               isSearchable={isSearchable === undefined ? isMulti : isSearchable}
               isClearable={isClearable}
               isMulti={isMulti}
+              isDisabled={isDisabled}
```

Comment:
I think this can also go on the FormControl^, unless our [chakra-react-select](https://github.com/csandman/chakra-react-select/tree/v3#isinvalid--default-false:~:text=You%20can%20pass%20also%20pass%20isInvalid%20or%20isDisabled%20to%20a%20wrapping%20%3CFormControl%20/%3E%20and%20it%20will%20output%20their%20corresponding%20%3CInput%20/%3E%20on%20the%20select) isn't hooked up right.

Reply:
Not hooked up correctly. When applied to the FormControl it doesn't disable the Chakra Select component.